esnext-browsers.js

```diff
@@ -117,6 +117,11 @@ module.exports = {
       short: 'FF 39',
       obsolete: true,
   },
+  firefox40: {
+    full: 'Firefox',
+      short: 'FF 40',
+      obsolete: true,
+  },
```

Comment:
I need it here https://github.com/kangax/compat-table/pull/853/files#diff-be5c570f292ab8dcff4e5c9e9a7cff9dR1052